internal/index/indexscanner/testdata/testindex/receipts/default-explicit.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: default-explicit-plugin
+spec:
+  platforms:
+  - bin: kubectl-test-plugin
+    files:
```

Comment:
Lets remove this as it has a default value and we aren’t testing this field’s parsed value.